src/crewai/cli/constants.py

```diff
@@ -62,6 +62,10 @@
             "prompt": "Enter your AWS Region Name (press Enter to skip)",
             "key_name": "AWS_REGION_NAME",
         },
+        {
+            "prompt": "Enter your AWS Bedrock API Key (30-day key from AWS console, press Enter to skip)",
+            "key_name": "AWS_BEARER_TOKEN_BEDROCK",
+        },
```

Comment:
The `AWS_BEARER_TOKEN_BEDROCK` is optional, and would be an alternative to providing the access key, secrety key etc